node/src/service/service_parachain.rs

```diff
@@ -138,8 +143,9 @@ where
 
             Ok(NimbusConsensus::build(BuildNimbusConsensusParams {
                 para_id: parachain_id,
+                backend,
                 proposer_factory,
-                block_import: client.clone(),
```

Comment:
[This](https://github.com/moonbeam-foundation/moonbeam/blob/7f85ea4d5f6feb9e1f37355c8269ffebc533dd51/node/service/src/lib.rs#L887) is what moonbeam uses for the block_import.

So moonbeam also didn't change this.

Reply:
https://github.com/zeitgeistpm/zeitgeist/pull/1059/commits/d015f8d01b9a5c7f5c4623948b7d0cb1c5bb1cb7